src/query/validation.c

```diff
@@ -237,7 +237,7 @@ static void verify_bucket_expression(Node *node)
     OpExpr *op_expr = (OpExpr *)node;
     FAILWITH_LOCATION(op_expr->location, "Use of operators to define buckets is not supported.");
   }
-  else if (is_simple_constant(node))
+  else if (is_stable_expression(node))
   {
     FAILWITH_LOCATION(exprLocation(node), "Simple constants are not allowed as bucket expressions.");
```

Comment:
Should we update error messages as well?

Reply:
Stable expression would be hard to decipher, it is after all pg internal stuff. I think "simple constant" works better on the outside, even despite being a little vague and intuitive.